packages/commonwealth/client/scripts/views/components/NewThreadForm/NewThreadForm.tsx

```diff
@@ -241,6 +283,37 @@ export const NewThreadForm = () => {
             )}
           </div>
         </div>
+        {featureFlags.gatingEnabled &&
+          showGatingBanner &&
+          restrictedTopicIds.includes(threadTopic.id) && (
+            <div className="gatingBanner">
+              <CWBanner
```

Comment:
We can make this banner a single component since it's used in multiple places -- for the gated group labels we can make a prop.